config/environments/production.rb

```diff
@@ -1,3 +1,5 @@
+require "active_support/core_ext/integer/time"
```

Comment:
whoops

Reply:
I'm not sure why that was added; can be removed right?

Reply:
yes

Reply:
As long as `1.hour.to_i` is no where in the file